public/star-filled.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Notka na przyszłość odnośnie pracy z assetami.
SVG (oraz inne typy obrazków ;) ) warto minimalizować/stripować z niepotrzebnych rzeczy. Do svg z mojego doświadczenia mogę polecić libke: https://www.npmjs.com/package/svgo
Możliwe są 2 podejścia:
1. instalujesz sobie svgo lokalnie a potem pamiętasz żeby obrazek przed wrzuceniem potraktować tym
2. dodajesz to dev depsów i jakoś wpinasz w proces budowania projektu (do `package.json` do skryptu można dodać, lub jakiś grunt/gulp, może nawet by się dało w dockerze, to już wedle preferencji)